examples/galaxies/plot_demographics.py

```diff
@@ -0,0 +1,239 @@
+logphi_massq = np.histogram(logm_massq, bins=bins)[0] / dV / dlm
+logphi_satq = np.histogram(logm_satq, bins=bins)[0] / dV / dlm
+
+logphi_active = logphi_centrals + logphi_satellites
+logphi_passive = logphi_massq + logphi_satq
+logphi_total = logphi_active + logphi_passive
+
+
+# %%
+# Validation against SSD DR7 data
+# -------------------------------
+#
+# Here we compare our sampled galaxies and
+# we validate the model using the results from the best fit to
+# SDSS DR7 data in Weigel et al. (2016) [2].
+# The authors presented a comprehensive method to determine
+# stellar mass functions and apply it to samples in the local universe,
+# in particular to SDSS DR7 data in the redshift range from 0.02 to 0.06.
+#
+# Their data is presented as logarithmic distribution, therefore,
+# to perform the comparison we need to apply a conversion factor.
+# This factor allows us to go from :math:`\phi` to a :math:`\log \phi` plot
+# and compare with Weigel et al 2016 best-fit model. 
+
+
+# Load the data
+wtotal = Table.read('weigel16_total.csv', format='csv')
+wred = Table.read('weigel16_quiescent.csv', format='csv')
+wblue = Table.read('weigel16_active.csv', format='csv')
+wcentral = Table.read('weigel16_central.csv', format='csv')
+wsatellite = Table.read('weigel16_satellite.csv', format='csv')
+
+# %%
+
+
+# Plot
+fig, (ax1, ax2, ax3) = plt.subplots(nrows=1, ncols=3, figsize=(16, 6), sharex=True, sharey=True)
+fig.suptitle('Galaxy Demographics Simulation', fontsize=26)
+
+ax1.plot(wblue['logm'], 10**wblue['logphi'], color='k', label='Weigel+16 active', lw=1)
+ax1.step(bins[:-1], logphi_active, where='post', label='SkyPy active', color='blue', zorder=3, lw=1)
+ax1.step(bins[:-1], logphi_centrals, where='post', label='SkyPy centrals', color='royalblue', zorder=3, lw=1)
+ax1.step(bins[:-1], logphi_satellites, where='post', label='SkyPy satellites', color='cyan', zorder=3, lw=1)
+
+
+ax2.plot(wred['logm'], 10**wred['logphi'], color='k', label='Weigel+16 passive', lw=1)
+ax2.fill_between(wred['logm'], 10**wred['upper_error'], 10**wred['lower_error'], color='salmon', alpha=0.1)
+ax2.step(bins[:-1], logphi_passive, where='post', label='SkyPy passive', color='red', zorder=3, lw=1)
+ax2.step(bins[:-1], logphi_massq, where='post', label='SkyPy mass-quenched', color='coral', zorder=3, lw=1)
+ax2.step(bins[:-1], logphi_satq, where='post', label='SkyPy sat-quenched', color='maroon', zorder=3, lw=1)
+
+ax3.plot(wtotal['logm'], 10**wtotal['logphi'], color='k', label='Weigel+16 total', lw=1)
+ax3.plot(wcentral['logm'], 10**wcentral['logphi'], '--', color='grey', label='Weigel+16 centrals', lw=1)
+ax3.plot(wsatellite['logm'], 10**wsatellite['logphi'], '--', color='grey', label='Weigel+16 satellites', lw=1)
+ax3.fill_between(wtotal['logm'], 10**wtotal['upper_error'], 10**wtotal['lower_error'], color='plum', alpha=0.1)
+ax3.step(bins[:-1], logphi_total, where='post', label='SkyPy total', color='purple', zorder=3, lw=1)
```

Comment:
Could you add line breaks here such that the user does not need to scroll horizontally?